CondFormats/DataRecord/interface/HGCalModuleConfigurationRcd.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CondFormats_HGCalModuleConfigurationRcd_h
+#define CondFormats_HGCalModuleConfigurationRcd_h
+// -*- C++ -*-
+//
+// Package:     CondFormats/DataRecord
+// Class  :     HGCalModuleConfigurationRcd
+//
+/**\class HGCalModuleConfigurationRcd HGCalModuleConfigurationRcd.h CondFormats/DataRecord/interface/HGCalModuleConfigurationRcd.h
+ *
+ * Description:
+ *   This record is used for passing the configuration parameters to the calibration step in RAW -> RECO,
+ *   This record depends on the HGCalMappingModuleIndexerRcd.
+ *
+ */
+//
+// Author:      Pedro Da Silva, Izaak Neutelings
+// Created:     Mon, 29 May 2023 09:13:07 GMT
+//
+
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "FWCore/Utilities/interface/mplVector.h"
+#include "CondFormats/DataRecord/interface/HGCalElectronicsMappingRcd.h"
+
+class HGCalModuleConfigurationRcd
+    : public edm::eventsetup::DependentRecordImplementation<HGCalModuleConfigurationRcd,
+                                                            edm::mpl::Vector<HGCalElectronicsMappingRcd> > {};
```

Comment:
We prefer to keep the bookkeeping of mapping/indexing separate from electronics configuration & calibration constants. In system tests the electronics configuration can vary between runs, for example for system tests and scans, while the geometry/indexing will stay constant. The idea is therefore to keep them separate because of different IOVs.